src/index.html

```diff
@@ -11,7 +11,205 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      id="form"> 
+      <fieldset class="fieldSet">
+        <legend>Personal Information</legend>
+     
+      
+        <div>
```

Comment:
```suggestion
        <legend>Personal Information</legend>
     
        <div>
```
Don't leave a lot of newlines